.github/workflows/mvn-verify-check.yml

```diff
@@ -167,16 +148,39 @@ jobs:
             fi
           }
           done
-
-  set-scala213-versions:
```

Comment:
github commit suggestion got triggered twice and deleted the an extra line
https://github.com/NVIDIA/spark-rapids/pull/11231/commits/b19a8f2e17801dbc7fd8bd2bceb4941d23d6a5be

Reply:
reverted the line